docs/how-to/GettingStarted.md

```diff
@@ -9,8 +9,9 @@ To setup your own MLOPs project in you azure subscription, follow these steps:
 - **[AML-related variables](../../configuration/configuration-aml.variables.yml)**: contains the definition of AML-related environment variables
 
 ```
-SDK_VERSION: the version of Azure ML SDK. Default value is 3.7
-AML_DATASET: training dataset name. Default value is 1.27
+PYTHON_VERSION: the version of python. Default value is 3.7
+SDK_VERSION: Default value is 1.27
```

Comment:
Please add here the description: "the version of Azure ML SDK. Default value is..."